mmengine/visualization/vis_backend.py

```diff
@@ -610,3 +601,89 @@ def close(self):
         """close an opened tensorboard object."""
         if hasattr(self, '_tensorboard'):
             self._tensorboard.close()
+
+
+@VISBACKENDS.register_module()
+class ClearmlVisBackend(BaseVisBackend):
+
```

Comment:
Please referred to https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/clearml.py to add docstring